particles/__init__.py

```diff
@@ -220,8 +220,11 @@ def update(self, force_print=False):
             'energy_density': self.energy_density
         })
 
+        #if force_print or self.regime != oldregime or self.in_equilibrium != oldeq:
+        #    print(self)
+
         if force_print or self.regime != oldregime or self.in_equilibrium != oldeq:
-            print(self)
+           print("\n"+ "\t" * 2 + "%s decoupled at T_dec = %.2f MeV \n" % (self.name, self.decoupling_temperature / UNITS.MeV) + ("\t" * 2 + "-" * 50))
```

Comment:
Hey, also there's a 3-space indentation. Do you mind installing something Python-specific instead of gedit? I use Sublime Text 3, but PyCharm might be even better but heavier

Reply:
That one was my fault. Standard indentation is set to 4 spaces.